woocommerce/woocommerce
# Add/33 two column product page layout

### All Submissions:






### Changes proposed in this Pull Request:

This adds several components to handle the product edit page. It is really simplistic as of now, and I think we should add slotfills within the `ProductFieldLayout` and `ProductFormLayout` at the end, so 3PD's can easily hook in with extra fields.

I am not set on the the implementation quite yet, and want to figure out a nicer way to pass fields in for each category. Also bear in mind that a category could have fields with an image on the side see the shipping dimensions section for example:
<img width="425" alt="Screen Shot 2022-07-28 at 4 42 41 PM" src="https://user-images.githubusercontent.com/2240960/181623868-9185f641-4311-43d4-a9a1-a4e155b3dee7.png">


Here is a quick demo of how it looks now:
<img width="1127" alt="Screen Shot 2022-07-28 at 4 33 17 PM" src="https://user-images.githubusercontent.com/2240960/181623927-3f7a3b4d-d9da-4e0d-ac98-5f5afcc04af0.png">

https://github.com/woocommerce/woocommerce/issues/34029

### How to test the changes in this Pull Request:

1. Enable the `new-product-management-experience` feature flag by using the WCA Test Helper (Tools > WCA Test Helper > Features)
2. Refresh the page and go to **Products > Add New (MVP)** and check if the layout matches that of the new design 5sAIeTRd9Yp7nSCT33BAWz-fi-5311%3A202025, fields don't need to match yet as those would be added in the future.
Also note that some of the `onChange` functions are empty callbacks, this is because the Form component isn't hooked up yet, this relies on https://github.com/woocommerce/woocommerce/pull/34082

### Other information:




### FOR PR REVIEWER ONLY:
